import_off.py

```diff
@@ -86,7 +86,7 @@ class ImportOFF(bpy.types.Operator, ImportHelper):
     def execute(self, context):
         #from . import import_off
 
-        keywords = self.as_keywords(ignore=('axis_forward',
+        keywords: self.as_keywords(ignore=('axis_forward',
```

Comment:
This is supposed to be an assignment, not a property/annotation. Making it an annotation breaks imports with an `UnboundLocalError` 8 lines down.